controllers/workspace/env/common_env_vars.go

```diff
@@ -20,40 +20,17 @@ import (
 func CommonEnvironmentVariables(workspaceName, workspaceId, namespace, creator string) []corev1.EnvVar {
 	return []corev1.EnvVar{
 		{
-			Name: "CHE_MACHINE_TOKEN",
-		},
-		{
-			Name:  "CHE_PROJECTS_ROOT",
-			Value: config.DefaultProjectsSourcesRoot,
-		},
-		{
-			Name:  "CHE_API",
-			Value: config.DefaultApiEndpoint,
-		},
-		{
-			Name:  "CHE_API_INTERNAL",
-			Value: config.DefaultApiEndpoint,
-		},
-		{
-			Name:  "CHE_API_EXTERNAL",
-			Value: config.DefaultApiEndpoint,
+			Name:  "DEVWORKSPACE_NAMESPACE",
+			Value: namespace,
```

Comment:
This is kind of redundant since namespace can be read from `/var/run/secrets/kubernetes.io/serviceaccount/namespace`, but I see how this would be useful so :+1:

Reply:
That's true that it duplicates already available info, but typically it's easier to read env var than file, it's why I added it as well, assuming that it should not hurt anyone